.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,96 @@
+        * [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
+        * [Registry](https://registry.terraform.io/) issues
+        * Spans resources across multiple providers
+
+        If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
+  - type: textarea
+    attributes:
+      label: Community Note
+      description: Please leave this note as is for the community
+      value: |
+        * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        * Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+        * If you are interested in working on this issue or have submitted a pull request, please leave a comment
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Terraform CLI and Terraform AWS Provider Version
+      description: |
+        Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed.
+
+        [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions).
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Affected Resource(s)
+      description: Please list the affected resource(s) and/or data source(s).
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: What should have happened?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual Behavior
+      description: What actually happened?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Terraform Configuration Files
+      description: |
+        Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.
+
+        For large Terraform configurations, please use a service like Dropbox and share a link to the ZIP file. For your security you can also encrypt the files using our [GPG public key](https://keybase.io/hashicorp).
+
+        [Information about code formatting](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Debug Output
```

Comment:
Currently we typically don't get `debug.log` for a lot of issues (though we probably should). Do we think this requirement will be a barrier?

Reply:
I went back and marked this as optional so as to not cause friction :+1: